Package.swift

```diff
@@ -19,15 +19,15 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/amplitude/analytics-connector-ios.git", from: "1.2.3")
+        .package(url: "https://github.com/amplitude/analytics-connector-ios.git", from: "1.0.1")
```

Comment:
1.2.3 -> 1.0.1? Feels like a pretty big jump?

Reply:
this was a [revert](https://github.com/amplitude/Amplitude-Swift/commit/61e77a16bc164f7eba86db35de2dcd308f828d9f) of all the changes to this file - could also go to `1.0.3` which has a few changes, or just leave it at latest.  release notes: https://github.com/amplitude/analytics-connector-ios/blob/main/CHANGELOG.md